src/package_tree.rs

```diff
@@ -419,3 +419,116 @@ pub fn get_package_name(path: &str) -> String {
     let bsconfig = read_bsconfig(&path);
     bsconfig.name
 }
+
+
+pub fn valide_package_dependencies(packages : & AHashMap<String, Package>) -> bool {
+    for (package_name, package) in packages {
+        let bs_dependencies = &package
+        .bsconfig.bs_dependencies.to_owned().unwrap_or(vec![]);
```

Comment:
Atm I'm just checking the `bs-dependencies`. We could extend this also to the `pinned-dependencies` or the `bs-dev-dependencies` 👍